src/views/404.js

```diff
@@ -1,9 +1,27 @@
-import React from 'react'
+import React from 'react';
+import Logo from '../assets/VeriFactLogo.svg';
 
-export default function NotFound (props) {
-  return (
-    <div>
-      <h1>404 page</h1>
-    </div>
-  )
+export default function NotFound(props) {
+	return (
+		<div
+			style={{
+				display: 'flex',
+				justifyContent: 'center',
+				flexDirection: 'column',
+				textAlign: 'center',
+				alignItems: 'center',
+				height: '100vh'
+			}}
+		>
+			<h1 style={{ fontSize: '6rem' }}>
+				Error 404! <br />Page Not Found!
```

Comment:
I think this might look a little cleaner with just one line of text. What do you think about removing the `Error 404!` text to keep things simple? Could also run this one by another designer if you'd like.